cub/cmake/CubInstallRules.cmake

```diff
@@ -16,7 +16,24 @@ install(DIRECTORY "${CUB_SOURCE_DIR}/cub/cmake/"
 )
 # Need to configure a file to store the infix specified in
 # CMAKE_INSTALL_INCLUDEDIR since it can be defined by the user
-set(install_location "${CMAKE_INSTALL_LIBDIR}/cmake/cub")
+set(_CCCL_RELATIVE_LIBDIR "${CMAKE_INSTALL_LIBDIR}")
+if(_CCCL_RELATIVE_LIBDIR MATCHES "^${CMAKE_INSTALL_PREFIX}")
+  # libdir is an abs string that starts with prefix
+  string(LENGTH "${CMAKE_INSTALL_PREFIX}" to_remove)
+  string(SUBSTRING "${_CCCL_RELATIVE_LIBDIR}" ${to_remove} -1 relative)
+  # remove any leading "/""
+  string(REGEX REPLACE "^/(.)" "\\1" _CCCL_RELATIVE_LIBDIR "${relative}")
+else()
+  message(FATAL_ERROR "CMAKE_INSTALL_LIBDIR must be a relative path or an absolute path under CMAKE_INSTALL_PREFIX")
```

Comment:
Let's update these error messages to print those variables out. CI is currently failing with this error and it'd be helpful to see what values its using.

Reply:
Done